docs/_layouts/default.html

```
@@ -0,0 +1,185 @@
+
+    <!-- GRID AREA: RIGHT NAV -->
+    <div id="right">
+      <h3>Search</h3>
+      <!-- Search Google -->
+      <div class="search-container" style="text-align: center; padding: 10px 0;">
+        <form method="get" action="https://www.google.com/custom" target="_blank">
+          <a href="https://www.google.com/">
+            <img src="https://www.google.com/logos/Logo_25wht.gif" alt="Google" style="border:0; vertical-align: middle; width: 100px;">
+          </a>
+          <br><br>
+          <input type="text" name="q" size="15" maxlength="255" placeholder="Search ASIMP..." style="width: 100%; padding: 6px; border: 1px solid var(--lab-border); border-radius: 6px; background: var(--lab-bg); color: var(--lab-text); box-sizing: border-box;">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)